### PR TITLE
Merge 2017-02-15

### LINKED ISSUE #373: a2ps: cannot open file `/home/linuxbrew/.linuxbrew/etc/a2ps.cfg'
# Please follow the general troubleshooting steps first:
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [x] If you're seeing permission errors tried running `sudo chown -R $(whoami) $(brew --prefix)`?

_You can erase any parts of this template not applicable to your Issue._
### Bug reports:

`a2ps source_file.java` fails if it is built from a bottle and relocated. However, it works fine if built from source.

Error message:

```
a2ps: cannot open file `/home/linuxbrew/.linuxbrew/etc/a2ps.cfg': No such file or directory
a2ps: cannot find file `level1.ppd': No such file or directory
```

`a2ps` is `cellar :any_skip_relocation`, but from what I've observed it should probably not do that.

### PR DESCRIPTION
**Pull request by rwhogg**
Closes #373

### CODE REVIEW

File: Formula/astyle.rb [hunk 1]
```diff
@@ -15,7 +15,6 @@ class Astyle < Formula
     sha256 "00b19ebc8e22131555090f45a62fe8e47bc147cbb6bd20b46126f64ececff0ba" => :sierra
     sha256 "7e80942d28e0b8767f65e4f90c0b783ffaebb5ed31ffc26278ebaf3c50563ca8" => :el_capitan
     sha256 "a3b000b925c1585ef9a5770da73f99a12e650d820a29a482afbe79838f599309" => :yosemite
-    sha256 "cf7468ff706254bf74edc9caab71a63850d49f45c18422b301dcdb9179565168" => :x86_64_linux
```
Comment (rwhogg) [inline Formula/astyle.rb]: Oops...

**Comment by rwhogg** [comment]
https://travis-ci.org/Linuxbrew/homebrew-core/builds/202141994#L8611
```
make -C src -f posix.mak
make[1]: Entering directory `/tmp/dmd-20170216-1136-i9x2eb/dmd-2.073.1-b2/src'
CC=c++ /tmp/.host_dmd-2.068.2/dmd2/linux/bin64/dmd -conf=/tmp/.host_dmd-2.068.2/dmd2/linux/bin64/dmd.conf idgen.d
  (CC)  ROOT_OBJS  root/newdelete.c
c++ -c -Wno-deprecated -Wstrict-aliasing -fno-exceptions -fno-rtti -D__pascal= -DMARS=1 -DTARGET_LINUX=1 -DDM_TARGET_CPU_X86=1 -m64 -std=gnu++98 -O2 -Iroot -MMD -MF newdelete.deps root/newdelete.c
/home/linuxbrew/.linuxbrew/Cellar/gcc/5.3.0/libexec/gcc/x86_64-unknown-linux-gnu/5.3.0/cc1plus: error while loading shared libraries: libz.so.1: cannot open shared object file: No such file or directory
make[1]: *** [newdelete.o] Error 1
make[1]: *** Waiting for unfinished jobs....
/home/linuxbrew/.linuxbrew/bin/ld: error while loading shared libraries: libz.so.1: cannot open shared object file: No such file or directory
collect2: error: ld returned 127 exit status
--- errorlevel 1
make[1]: *** [idgen] Error 1
make[1]: Leaving directory `/tmp/dmd-20170216-1136-i9x2eb/dmd-2.073.1-b2/src'
make: *** [all] Error 2
```
A number of formulae also had "C compiler could not create executables, like `gputils` and `libquvi`.

**Comment by sjackman** [comment]
> `cc1plus: error while loading shared libraries: libz.so.1: cannot open shared object file`

This error is a bug in Linuxbrew/test-bot TravisCi. I've reproduced it in https://github.com/Linuxbrew/homebrew-core/pull/1710

**Comment by sjackman** [comment]
Here's a hopeful fix: https://github.com/Linuxbrew/homebrew-core/pull/1806

**Pull request merged**